exo/networking/manual/read_config.py

```diff
@@ -0,0 +1,113 @@
+from exo.topology.device_capabilities import DeviceCapabilities, DeviceFlops
+import socket
+import yaml
+
+class ReadManualConfig():
```

Comment:
Why is this a class? Just make a single function that reads the device capabilities from the file

Reply:
Frankly, I think it is more flexible.
For exemple, for now, I set args.wait_for_peers = 1 in main.py.
But after we finish debug, I want the number of peer to wait to comme from ReadManualConfig.

Reply:
I dont understand what you mean